app/assets/stylesheets/recipient_form.css.scss

```diff
@@ -4,7 +4,7 @@ $recipient-image-height: 500px;
     .recipient-form-image {
         z-index: -20;
         background-size: cover;
-        background-image: url("landing_splash_berries.jpg");
+        background-image: url("https://s3-us-west-2.amazonaws.com/foodshift/assets/images/landing_splash_berries.jpg");
```

Comment:
URL `https://s3-us-west-2.amazonaws.com/foodshift/assets/images/landing_splash_berries.jpg` should not contain protocol or domain